framework/include/media/BufferOutputDataSource.h

```diff
@@ -100,8 +100,18 @@ class BufferOutputDataSource : public OutputDataSource
 	 */
 	bool close() override;
 
-protected:
-	ssize_t onStreamBufferReadable(bool isFlush) override;
+	/**
+	 * @brief Write the file
+	 * @details @b #include <media/BufferOutputDataSource.h>
+	 * param[in] buf poiter to a buffer
+	 * param[in] size Number of size to write
+	 * @return True.
```

Comment:
It seems that "True" needs to be changed as something like "Data size written to buf".

Reply:
Fixed it